### ISSUE TITLE
[BUG] - Separate languages only working in chapters list

### ISSUE THREAD

**Issue by MaxEhrhart** [Glagan/SyncDex#41]
**Describe the bug**
The 'separate in languages' feature is working only in history page, 
I don't know if the removal was intentional from the manga page or not, before one of the updates it worked, don't remember which one though.
If it was intentional, could you add an option in the config to enable it?
It was a great feature :)

**Screenshots**
Configs:
![image](https://user-images.githubusercontent.com/22263628/106392399-6132d200-63d0-11eb-9595-c680e775b2bf.png)

History Page
![image](https://user-images.githubusercontent.com/22263628/106392414-70198480-63d0-11eb-9710-59fd21210166.png)

Manga page
![image](https://user-images.githubusercontent.com/22263628/106392153-5d528000-63cf-11eb-97e0-d00ce48be915.png)

**Browser**
Chrome 88.0.4324.104 64 bits

**Options and logs**
```
{"logs":[{"d":1609473337034,"msg":"Updating from version 0.1.8 to 0.1.9"},{"d":1609787914189,"msg":"Error on request [https://mangadex.org/api/v2/user/me/manga/59539]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1609864040204,"msg":"Error on request [https://mangadex.org/api/v2/user/me/manga/47818]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1610407022434,"msg":"Updating from version 0.1.9 to 0.2.0"},{"d":1610410247548,"msg":"MyAnimeList::get ~ Execution time: 826.600000000326ms"},{"d":1610410248527,"msg":"Anilist::get ~ Execution time: 1805.7150000022375ms"},{"d":1610410256700,"msg":"Anilist::get ~ Execution time: 5197.605000001204ms"},{"d":1610410347044,"msg":"MyAnimeList::get ~ Execution time: 936.7700000002515ms"},{"d":1610410348202,"msg":"Anilist::get ~ Execution time: 2095.3000000008615ms"},{"d":1610410396031,"msg":"Anilist::get ~ Execution time: 790.660000006028ms"},{"d":1610450996248,"msg":"MyAnimeList::get ~ Execution time: 436.01000000489876ms"},{"d":1610450997366,"msg":"Anilist::get ~ Execution time: 1554.359999994631ms"},{"d":1610451006151,"msg":"MyAnimeList::get ~ Execution time: 836.555000001681ms"},{"d":1610451011425,"msg":"MyAnimeList::get ~ Execution time: 1497.815000009723ms"},{"d":1610451012658,"msg":"Anilist::get ~ Execution time: 2731.0900000011316ms"},{"d":1610451317922,"msg":"MyAnimeList::get ~ Execution time: 1066.6849999979604ms"},{"d":1610451318382,"msg":"Anilist::get ~ Execution time: 1526.9700000062585ms"},{"d":1610451328181,"msg":"MyAnimeList::get ~ Execution time: 6726.509999993141ms"},{"d":1610451329303,"msg":"Anilist::get ~ Execution time: 7846.124999996391ms"},{"d":1610451332581,"msg":"MyAnimeList::get ~ Execution time: 9805.319999999483ms"},{"d":1610451333223,"msg":"Anilist::get ~ Execution time: 10446.909999998752ms"},{"d":1610451390744,"msg":"MyAnimeList::get ~ Execution time: 884.1150000080233ms"},{"d":1610451392037,"msg":"Anilist::get ~ Execution time: 2177.090000011958ms"},{"d":1610451440172,"msg":"MyAnimeList::get ~ Execution time: 942.2999999951571ms"},{"d":1610451441474,"msg":"Anilist::get ~ Execution time: 2244.860000006156ms"},{"d":1610451517395,"msg":"MyAnimeList::get ~ Execution time: 529.1500000021188ms"},{"d":1610451518671,"msg":"Anilist::get ~ Execution time: 1805.529999997816ms"},{"d":1610504130079,"msg":"MyAnimeList::get ~ Execution time: 516.0850000102073ms"},{"d":1610504131178,"msg":"Anilist::get ~ Execution time: 1616.859999980079ms"},{"d":1610507209279,"msg":"MyAnimeList::get ~ Execution time: 436.22999999206513ms"},{"d":1610507210528,"msg":"Anilist::get ~ Execution time: 1686.0049999959301ms"},{"d":1610538844611,"msg":"MyAnimeList::get ~ Execution time: 2241.9599999848288ms"},{"d":1610538845112,"msg":"Anilist::get ~ Execution time: 2743.6450000095647ms"},{"d":1610538847912,"msg":"Anilist::get ~ Execution time: 5202.6500000210945ms"},{"d":1610538862767,"msg":"MyAnimeList::get ~ Execution time: 4590.010000014445ms"},{"d":1610538863917,"msg":"Anilist::get ~ Execution time: 5741.25500000082ms"},{"d":1610538866616,"msg":"Anilist::get ~ Execution time: 7759.749999997439ms"},{"d":1610538869509,"msg":"MyAnimeList::get ~ Execution time: 9286.78999998374ms"},{"d":1610538870794,"msg":"Anilist::get ~ Execution time: 10571.624999982305ms"},{"d":1610538873566,"msg":"MyAnimeList::get ~ Execution time: 11954.544999985956ms"},{"d":1610538874717,"msg":"Anilist::get ~ Execution time: 13105.225000006612ms"},{"d":1610539105644,"msg":"MyAnimeList::get ~ Execution time: 488.38500000420026ms"},{"d":1610539106850,"msg":"Anilist::get ~ Execution time: 1694.2049999779556ms"},{"d":1610539335450,"msg":"Anilist::get ~ Execution time: 443.06999997934327ms"},{"d":1610539434013,"msg":"MyAnimeList::get ~ Execution time: 448.5399999830406ms"},{"d":1610539435257,"msg":"Anilist::get ~ Execution time: 1693.10500001302ms"},{"d":1610539636720,"msg":"MyAnimeList::get ~ Execution time: 341.51500000734814ms"},{"d":1610539637909,"msg":"Anilist::get ~ Execution time: 1531.1499999952503ms"},{"d":1610539648181,"msg":"MyAnimeList::get ~ Execution time: 603.2800000102725ms"},{"d":1610539649372,"msg":"Anilist::get ~ Execution time: 1794.9599999992643ms"},{"d":1610539656600,"msg":"MyAnimeList::get ~ Execution time: 4177.60999998427ms"},{"d":1610539657440,"msg":"Anilist::get ~ Execution time: 5017.805000010412ms"},{"d":1610539695981,"msg":"MyAnimeList::get ~ Execution time: 1330.6049999955576ms"},{"d":1610539696651,"msg":"Anilist::get ~ Execution time: 2016.475000011269ms"},{"d":1610592975974,"msg":"MyAnimeList::get ~ Execution time: 1978.6399999866262ms"},{"d":1610592978367,"msg":"Anilist::get ~ Execution time: 3760.884999996051ms"},{"d":1610592988941,"msg":"Anilist::get ~ Execution time: 7082.369999989169ms"},{"d":1610592995881,"msg":"MyAnimeList::get ~ Execution time: 10646.650000009686ms"},{"d":1610592997030,"msg":"Anilist::get ~ Execution time: 11796.440000005532ms"},{"d":1610592999950,"msg":"MyAnimeList::get ~ Execution time: 12031.584999989718ms"},{"d":1610593001201,"msg":"Anilist::get ~ Execution time: 13282.785000017611ms"},{"d":1610593233158,"msg":"MyAnimeList::get ~ Execution time: 434.78999999933876ms"},{"d":1610593234335,"msg":"Anilist::get ~ Execution time: 1612.7100000157952ms"},{"d":1610593338239,"msg":"MyAnimeList::get ~ Execution time: 450.59500000206754ms"},{"d":1610593339390,"msg":"Anilist::get ~ Execution time: 1601.3749999983702ms"},{"d":1610593484544,"msg":"Anilist::get ~ Execution time: 725.1699999906123ms"},{"d":1610593555620,"msg":"MyAnimeList::get ~ Execution time: 830.1700000010896ms"},{"d":1610623106967,"msg":"Updating from version 0.2.0 to 0.2.1"},{"d":1610623106977,"msg":"Applied patch version 0.2.0.1"},{"d":1610659491611,"msg":"Updating from version 0.2.1 to 0.2.2"},{"d":1610908663177,"msg":"Error on request [https://mangadex.org/api/v2/user/me/manga/60450]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1611019354099,"msg":"Updating from version 0.2.2 to 0.2.3"},{"d":1611019427467,"msg":"Found MangaDex max Chapter undefined"},{"d":1611019433294,"msg":"Volume reset chapters ? false"},{"d":1611019582395,"msg":"Found MangaDex max Chapter undefined"},{"d":1611019583235,"msg":"Volume reset chapters ? false"},{"d":1611020637337,"msg":"Found MangaDex max Chapter undefined"},{"d":1611020638649,"msg":"Volume reset chapters ? false"},{"d":1611020645602,"msg":"Volume reset chapters ? false"},{"d":1611020681879,"msg":"Found MangaDex max Chapter undefined"},{"d":1611020682741,"msg":"Volume reset chapters ? false"},{"d":1611020843100,"msg":"Volume reset chapters ? false"},{"d":1611020843751,"msg":"Volume reset chapters ? false"},{"d":1611020884960,"msg":"Found MangaDex max Chapter undefined"},{"d":1611020886096,"msg":"Volume reset chapters ? false"},{"d":1611020895397,"msg":"Volume reset chapters ? false"},{"d":1611020923821,"msg":"Found MangaDex max Chapter undefined"},{"d":1611020924661,"msg":"Volume reset chapters ? false"},{"d":1611021383079,"msg":"Found MangaDex max Chapter undefined"},{"d":1611021384363,"msg":"Volume reset chapters ? false"},{"d":1611021404245,"msg":"Found MangaDex max Chapter undefined"},{"d":1611021405477,"msg":"Volume reset chapters ? false"},{"d":1611021412146,"msg":"Volume reset chapters ? false"},{"d":1611021463456,"msg":"Found MangaDex max Chapter undefined"},{"d":1611021464722,"msg":"Volume reset chapters ? false"},{"d":1611021959677,"msg":"Found MangaDex max Chapter undefined"},{"d":1611021963591,"msg":"Volume reset chapters ? false"},{"d":1611057911513,"msg":"Volume reset chapters ? false"},{"d":1611057944285,"msg":"Found MangaDex max Chapter undefined"},{"d":1611057945091,"msg":"Volume reset chapters ? false"},{"d":1611057953298,"msg":"Volume reset chapters ? false"},{"d":1611104256318,"msg":"Volume reset chapters ? false"},{"d":1611104700383,"msg":"Found MangaDex max Chapter undefined"},{"d":1611104701748,"msg":"Volume reset chapters ? false"},{"d":1611104704701,"msg":"Volume reset chapters ? false"},{"d":1611104709335,"msg":"Found MangaDex max Chapter undefined"},{"d":1611104711444,"msg":"Volume reset chapters ? false"},{"d":1611104728536,"msg":"Found MangaDex max Chapter undefined"},{"d":1611104729411,"msg":"Volume reset chapters ? false"},{"d":1611108824920,"msg":"Found MangaDex max Vol. 3 Ch. 23"},{"d":1611108829358,"msg":"Found MangaDex max Vol. 7 Ch. 72"},{"d":1611108835558,"msg":"Volume reset chapters ? false"},{"d":1611108840123,"msg":"Volume reset chapters ? false"},{"d":1611108841621,"msg":"Volume reset chapters ? false"},{"d":1611108850205,"msg":"Volume reset chapters ? false"},{"d":1611113278286,"msg":"Found MangaDex max Chapter undefined"},{"d":1611113280255,"msg":"Volume reset chapters ? false"},{"d":1611113313810,"msg":"Found MangaDex max Chapter undefined"},{"d":1611113315310,"msg":"Volume reset chapters ? false"},{"d":1611113322882,"msg":"Found MangaDex max Chapter undefined"},{"d":1611139978246,"msg":"Volume reset chapters ? false"},{"d":1611139983451,"msg":"Found MangaDex max Vol. 7 Ch. 72"},{"d":1611139984932,"msg":"Found MangaDex max Vol. 3 Ch. 23"},{"d":1611140636357,"msg":"Found MangaDex max Chapter undefined"},{"d":1611140638154,"msg":"Volume reset chapters ? false"},{"d":1611140722511,"msg":"Found MangaDex max Chapter undefined"},{"d":1611140723714,"msg":"Volume reset chapters ? false"},{"d":1611140755482,"msg":"Found MangaDex max Vol. 7 Ch. 72"},{"d":1611140757683,"msg":"Volume reset chapters ? false"},{"d":1611140780620,"msg":"Volume reset chapters ? false"},{"d":1611140785311,"msg":"Found MangaDex max Vol. 3 Ch. 51"},{"d":1611140807399,"msg":"Found MangaDex max Vol. 3 Ch. 23"},{"d":1611140873621,"msg":"Found MangaDex max Vol. 3 Ch. 51"},{"d":1611140875910,"msg":"Volume reset chapters ? false"},{"d":1611141081668,"msg":"Found MangaDex max Chapter undefined"},{"d":1611141083154,"msg":"Volume reset chapters ? false"},{"d":1611141117896,"msg":"Found MangaDex max Chapter 60"},{"d":1611141220753,"msg":"Found MangaDex max Chapter 60"},{"d":1611141221880,"msg":"Volume reset chapters ? false"},{"d":1611182947487,"msg":"Volume reset chapters ? false"},{"d":1611182951477,"msg":"Found MangaDex max Chapter undefined"},{"d":1611182953386,"msg":"Volume reset chapters ? false"},{"d":1611182986410,"msg":"Found MangaDex max Chapter undefined"},{"d":1611182988496,"msg":"Volume reset chapters ? false"},{"d":1611183006565,"msg":"Volume reset chapters ? false"},{"d":1611193431419,"msg":"Found MangaDex max Chapter 100"},{"d":1611193432829,"msg":"Volume reset chapters ? false"},{"d":1611194043417,"msg":"Volume reset chapters ? false"},{"d":1611194272579,"msg":"Found MangaDex max Chapter undefined"},{"d":1611194273767,"msg":"Volume reset chapters ? false"},{"d":1611194341812,"msg":"Found MangaDex max Chapter undefined"},{"d":1611194351308,"msg":"Found MangaDex max Vol. 9 Ch. 116"},{"d":1611194395461,"msg":"Found MangaDex max Vol. 9 Ch. 116"},{"d":1611194396377,"msg":"Volume reset chapters ? false"},{"d":1611194401196,"msg":"Found MangaDex max Chapter undefined"},{"d":1611194403133,"msg":"Volume reset chapters ? false"},{"d":1611194817512,"msg":"Found MangaDex max Chapter undefined"},{"d":1611194818822,"msg":"Volume reset chapters ? false"},{"d":1611227199288,"msg":"Volume reset chapters ? false"},{"d":1611227227789,"msg":"Volume reset chapters ? false"},{"d":1611227241069,"msg":"Found MangaDex max Chapter undefined"},{"d":1611227241949,"msg":"Volume reset chapters ? false"},{"d":1611227251206,"msg":"Volume reset chapters ? false"},{"d":1611227256128,"msg":"Found MangaDex max Vol. 2 Ch. 34"},{"d":1611227324587,"msg":"Found MangaDex max Chapter undefined"},{"d":1611227325840,"msg":"Volume reset chapters ? false"},{"d":1611227475231,"msg":"Found MangaDex max Vol. 2 Ch. 34"},{"d":1611227476358,"msg":"Volume reset chapters ? false"},{"d":1611314034543,"msg":"Found MangaDex max Chapter 15"},{"d":1611314036257,"msg":"Volume reset chapters ? false"},{"d":1611314060428,"msg":"Found MangaDex max Chapter 15"},{"d":1611314208076,"msg":"Found MangaDex max Chapter 15"},{"d":1611314209310,"msg":"Volume reset chapters ? false"},{"d":1611314216617,"msg":"Volume reset chapters ? false"},{"d":1611314220090,"msg":"Volume reset chapters ? false"},{"d":1611314226230,"msg":"Volume reset chapters ? false"},{"d":1611314226798,"msg":"Found MangaDex max Chapter 30"},{"d":1611314235697,"msg":"Volume reset chapters ? false"},{"d":1611314440889,"msg":"Found MangaDex max Chapter undefined"},{"d":1611314443364,"msg":"Volume reset chapters ? false"},{"d":1611314633619,"msg":"Found MangaDex max Chapter undefined"},{"d":1611314635357,"msg":"Volume reset chapters ? false"},{"d":1611314650283,"msg":"Found MangaDex max Chapter undefined"},{"d":1611314651115,"msg":"Volume reset chapters ? false"},{"d":1611314664836,"msg":"Found MangaDex max Chapter undefined"},{"d":1611314668521,"msg":"Volume reset chapters ? false"},{"d":1611314685025,"msg":"Found MangaDex max Chapter 30"},{"d":1611314686182,"msg":"Volume reset chapters ? false"},{"d":1611314871979,"msg":"Found MangaDex max Chapter undefined"},{"d":1611314873330,"msg":"Volume reset chapters ? false"},{"d":1611314879233,"msg":"Volume reset chapters ? false"},{"d":1611315018997,"msg":"Found MangaDex max Chapter undefined"},{"d":1611315020237,"msg":"Volume reset chapters ? false"},{"d":1611361075484,"msg":"Found MangaDex max Chapter 36"},{"d":1611361077074,"msg":"Volume reset chapters ? false"},{"d":1611415655990,"msg":"Volume reset chapters ? false"},{"d":1611416763347,"msg":"Found MangaDex max Chapter undefined"},{"d":1611416765431,"msg":"Volume reset chapters ? false"},{"d":1611416780964,"msg":"Volume reset chapters ? false"},{"d":1611416785705,"msg":"Volume reset chapters ? false"},{"d":1611416787028,"msg":"Volume reset chapters ? false"},{"d":1611416796257,"msg":"Volume reset chapters ? false"},{"d":1611416924941,"msg":"Found MangaDex max Chapter undefined"},{"d":1611416926428,"msg":"Volume reset chapters ? false"},{"d":1611417165199,"msg":"Found MangaDex max Chapter undefined"},{"d":1611417167143,"msg":"Volume reset chapters ? false"},{"d":1611417253439,"msg":"Found MangaDex max Chapter undefined"},{"d":1611417254985,"msg":"Volume reset chapters ? false"},{"d":1611417271867,"msg":"Found MangaDex max Chapter undefined"},{"d":1611417369617,"msg":"Volume reset chapters ? false"},{"d":1611417424251,"msg":"Found MangaDex max Chapter undefined"},{"d":1611417425486,"msg":"Volume reset chapters ? false"},{"d":1611446983753,"msg":"Volume reset chapters ? false"},{"d":1611447008787,"msg":"Found MangaDex max Chapter undefined"},{"d":1611447009714,"msg":"Volume reset chapters ? false"},{"d":1611505002745,"msg":"Updating from version 0.2.3 to 0.2.4"},{"d":1611523675353,"msg":"Updating from version 0.2.4 to 0.2.5"},{"d":1611702740951,"msg":"Updating from version 0.2.5 to 0.2.6"},{"d":1611791039341,"msg":"Updating from version 0.2.6 to 0.2.7"},{"d":1611961402180,"msg":"Updating from version 0.2.7 to 0.2.8"},{"d":1612112638463,"msg":"Updated from version 0.2.8 to 0.2.9"}],"options":{"acceptLowScore":false,"autoSync":true,"biggerHistory":true,"chapterStatus":false,"chaptersSaved":100,"checkOnStartup":false,"checkOnStartupCooldown":30,"checkOnStartupMainOnly":true,"colors":{"higherChapter":"transparent","highlights":["rgba(28, 103, 141, 0.8)"],"lowerChapter":"rgba(180, 102, 75, 0.4)","nextChapter":"rgba(199, 199, 199, 0.4)","openedChapter":"rgba(28, 135, 141, 0.4)"},"confirmChapter":true,"errorNotifications":true,"favoriteLanguage":"gb","groupTitlesInLists":true,"hideHigher":false,"hideLast":false,"hideLower":false,"highlight":true,"iconsSilentAfterSync":false,"linkToServices":true,"logLevel":0,"mainService":"mal","mdUpdateSyncDex":true,"noReloadStatus":true,"notifications":true,"originalThumbnail":false,"overviewMainOnly":true,"progressInThumbnail":true,"saveOnLastPage":true,"saveOnlyHigher":true,"saveOnlyNext":true,"saveOpenedChapters":true,"separateLanguages":true,"services":["mal","al"],"silentUpdate":false,"subVersion":9,"thumbnail":true,"thumbnailMaxHeight":80,"tokens":{"anilistToken":"set"},"updateMD":true,"updateMDProgress":true,"updateOnlyInList":true,"useMochi":true,"version":0.2}}
```

**Comment by Glagan** [comment]
No the option should still be available on chapter lists and on the title page, this looks like a bug.

**Comment by Glagan** [comment]
Fixed in version ``0.2.10``.
